### ISSUE TITLE
Including informative documents as a design principle

### ISSUE THREAD

**Issue by putzhuber** [w3c/silver#307]
### Question:
Are there additional Design Principles and Requirements that should be included in the WCAG 3 project?

### Suggested Design principles:
5. Accessibility guidelines should … be written in non technical  language, that is easy to translate and as easy as possible to understand.

??. Accessibility guidelines should … offer a structure of accompanying documents, that are clear and concise and allow to find technical information (methods, how-tos) quickly and in one place.

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by jspellman** [comment]
Draft response: We discussed your idea in our [meeting of 18 March 2022](https://www.w3.org/2022/03/18-silver-minutes.html#t01) and agreed with your ideas, but we thought that we had it covered in Design Principles 4, 5, and 10.  It is possible you overlooked it since there are a lot of Design Principles and your would need to see it in total.  We discussed rewriting the design principle, but decided that since it is a minor clarification change, we would rather spend our time working on WCAG3 itself, rather than refining the wording of the Requirements document.  Please let us know if you strongly disagree by filing another issue and referring to this one.

**Comment by alastc** [comment]
There is an underlying assumption that there will be informative documents, that is part of the whole.

The question is whether we have a requirement for informative documents?

Sub-group thoughts: No harm in adding a requiement for informative docs.

E.g. "There should be accompanying documents that are clear, concise, and findable technical information (methods, how-tos) quickly in one place."

**Comment by rachaelbradley** [comment]
Slight suggested rewording: "There should be accompanying documents that are clear, concise, and **provide** findable technical information (methods, how-tos) quickly in one place." OR "There should be accompanying documents **that provide** clear, concise, and findable technical information (methods, how-tos) quickly in one place."

**Comment by alastc** [comment]
Closing from catchup PR #736
